core/templates/pages/exploration-editor-page/settings-tab/templates/reassign-role-confirmation-modal.directive.html

```diff
@@ -0,0 +1,21 @@
+<div class="modal-header oppia-modal-header">
+  <h3>Are you sure?</h3>
+</div>
+
+<div class="modal-body oppia-modal-body">
+  <p>Are you sure you want to reassign <strong><[username]></strong> from <strong><[oldRole]></strong> to the role <strong><[newRole]></strong>?</p>
+</div>
+
+<div class="modal-footer">
+  <button class="btn btn-secondary" ng-click="cancel()">Cancel</button>
+  <button class="btn btn-danger protractor-test-really-delete-exploration-button" ng-click="confirm()">Reassign user</button>
+</div>
+
+<style>
+  .oppia-modal-header > h3 {
```

Comment:
You can directly use the class in h3, right?

Reply:
Also, "oppia-modal-header" seems too generic (we don't want to use it in other modals). Perhaps "reassign-role-confirmation-modal-header"?

Reply:
Done.